Makefile

```diff
@@ -12,17 +13,28 @@ all:
 
 sanity:
 	@for file in $$(find lib -type f -name "*.pm" | sort); do \
-		perl -cw -Ilib $$file || exit 1; \
+		$(PERL_BIN)/perl -cw -Ilib $$file || exit 1; \
 	done
 
 test: sanity
 	-$(MAKE) elevate-cpanel
-	/usr/local/cpanel/3rdparty/bin/prove t/00_load.t
-	/usr/local/cpanel/3rdparty/bin/yath test -j8 t/*.t
+	$(PERL_BIN)/prove t/00_load.t
+	$(PERL_BIN)/yath test -j8 t/*.t
+
+build:
+    curl -fsSL https://raw.githubusercontent.com/skaji/cpm/main/cpm > /usr/sbin/cpm
+	chmod -v +x /usr/sbin/cpm
+	/scripts/update_local_rpm_versions --edit target_settings.perl-enhanced installed
+	cpm install --with-all --cpanfile t/cpanfile
```

Comment:
I think you're not specifying perl and going to end up with installing to system perl.

Reply:
Strange I had the `--global` in a previous commit and managed to yank it.

```
    -g, --global
          install modules into current @INC instead of local/
```

Reply:
What `cpm` will actually do is default to $PWD/local/lib/perl5/ as where everything is thrown.  

I changed this around as I really only needed to install Test/PerlTidy.pm so I used that $PWD/local/lib/perl5/ path for the copy and then immediately removed $PWD/local/ as that being in place completely obliterates the yath tests.